contrib/kvm/create-instance-kvm

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import time
+import libvirt
+from libvirt import libvirtError
+xmlconfig = """
+<domain type='kvm' id='1'>
+    <name>ubuntu-14.04.5-desktop-unattended</name>
+    <memory unit='KiB'>1048576</memory>
+    <currentMemory unit='KiB'>1048576</currentMemory>
+    <vcpu placement='static'>1</vcpu>
+    <resource>
+        <partition>/machine</partition>
+    </resource>
+    <os>
+        <type arch='x86_64' machine='pc-i440fx-trusty'>hvm</type>
+        <boot dev='hd'/>
+        <boot dev='cdrom'/>
+    </os>
+    <features>
+        <acpi/>
+        <apic/>
+        <pae/>
+    </features>
+    <clock offset='utc'/>
+    <on_poweroff>destroy</on_poweroff>
+    <on_reboot>restart</on_reboot>
+    <on_crash>restart</on_crash>
+    <devices>
+        <emulator>/usr/bin/qemu-system-x86_64</emulator>
+        <disk type='file' device='disk'>
+            <driver name='qemu' type='raw'/>
+            <source file='/var/lib/libvirt/images/ubuntu-14.04.5-desktop-amd64-unattended.qcow2'/>
+            <backingStore/>
+            <target dev='vda' bus='virtio'/>
+            <address type='pci' domain='0x0000' bus='0x00' slot='0x07' function='0x0'/>
+        </disk>
+        <disk type='file' device='cdrom'>
+            <driver name='qemu' type='raw'/>
+            <source file='/var/lib/libvirt/images/my-seed.img'/>
+            <backingStore/>
+            <target dev='hdb' bus='ide'/>
+            <address type='drive' controller='0' bus='0' target='0' unit='1'/>
+            <on_reboot>destroy</on_reboot>
+        </disk>
+        <disk type='file' device='cdrom'>
+            <driver name='qemu' type='raw'/>
+            <source file='/var/lib/libvirt/images/ubuntu-14.04.5-desktop-amd64-unattended.iso'/>
```

Comment:
trusty not xenial

Reply:
README means to run the host on Xenial and the guest os is Trusty. However I am +1 to migrate the guest to Xenial.

Reply:
I felt strange to see inconsistent versions in host and guest, but it doesn't really matter for runtime.  Bring them together is good, but if you don't want to spend that time, stating clearly the inconsistency in your readme works equally good.